src/duct_hierarchy.edn

```diff
@@ -0,0 +1 @@
+{:duct.server.http/http-kit [:duct.server/http]}
```

Comment:
Is it possible for you to amend the commit and add in an ending newline?

Reply:
done